CHANGELOG.md

```diff
@@ -1,9 +1,22 @@
-# dbt_amazon_ads_source v0.UPDATE.UPDATE
-
- ## Under the Hood:
-
+# dbt_amazon_ads_source v0.2.0
+## 🚨 Breaking changes
+- This release is labeled breaking to reflect the Fivetran Amazon Ads connector's upgrade from version 2 to 3 of the Sponsored Products API. Further details are also available in the [June 2023 release notes](https://fivetran.com/docs/applications/amazon-ads/changelog#june2023).
+## 🎉 Features
+[PR #11](https://github.com/fivetran/dbt_amazon_ads_source/pull/11) includes the following updates:
+- Added a new column, `native_language_locale`, to the `keyword_history` staging tables.
+- Made the following changes to the `campaign_history` table:
+  - Added three new columns, `budget`, `budget_type`, and `effective_budget`
+  - Removed the `campaign_type`, `daily_budget`, `placement`, and `premium_bid_adjustment` columns
```

Comment:
This is a breaking change. Would you be able to include this in the breaking section so customers are immediately aware of the change.

Reply:
updated

Reply:
I never really understand why GitHub highlights multiple lines in the review when I only selected one haha In my review I only meant this comment to pertain to line 9 in the above code block. The removal of fields are the breaking changes, but the others are more features and align with you previous CHANGELOG entry. 

Would you be able to only include the removed fields as the breaking change to make it very apparent to users what fields are being removed. The others may then be listed as Feature updates as you had previously. Apologies for the confusion.

Reply:
Oh got it! I have updated.